hdw.dat.cly

```diff
@@ -8,6 +8,10 @@
 #
 #  1   2      3        4         5       6      7      8    9   10   11   12  13(1)  13(2)  13(3)  14   15  16 17
 #
+# UNTIL 13/Jan/2020 00:00:00 Date that Borealis installed in Clyde
+   66 2020 1036800 70.487  -68.504  28.0  -55.62  3.24 1.0 10.0 0.0 1.0  0.0 100.0 0.0 0.0 0 100 16
+# UNTIL 01/Feb/2022 18:00:00.0 When Borealis changed to 3.24 degree beam separation
+   66 2022 2786400  70.487  -68.504  28.0  -55.62  3.5 1.0 10.0 0.0 1.0  0.0 100.0 0.0 0.0 0 100 16
```

Comment:
I think this `yrsec` corresponds to `20220202 06:00:00 UT`, rather than `20220201 18:00:00 UT` as the comment line indicates.  Should the `yrsec` instead be `2743200`?

Reply:
You're right, I was using timedelta incorrectly and assumed it would work the same pos and neg, but that's not correct:
```
>>> d
datetime.datetime(2022, 2, 1, 18, 0)
>>> d2
datetime.datetime(2022, 1, 1, 0, 0)
>>> d - d2
datetime.timedelta(31, 64800)
>>> d2 -d
datetime.timedelta(-32, 21600)
```